### PR TITLE
Awesome react gallery version 1

### PR DESCRIPTION
**Pull request by kunalpanchal**
- Setup README.md & .gitignore
- All config files added
- demo added. Run `npm i && npm run demo`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    );
+  }
+}
+```
+
+## Config
+
+| Key  | Purpose |
+| ------------- | ------------- |
+| server  | A url for the server APIs  |
```
Comment (shivgarg5676) [inline README.md]: Write about server url is required to upload images

File: dist/index.es.js [hunk 2]
```diff
@@ -0,0 +1,1665 @@
+import React, { Component } from 'react';
```
Comment (shivgarg5676) [inline dist/index.es.js]: Add dist folder to git ignore. not required here

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import babel from "rollup-plugin-babel";
```
Comment (shivgarg5676) [inline rollup.config.js]: any specific reason to use rollup instead of webpack to build

File: README.md (hunk 1 above)
Comment (shivgarg5676) [reply README.md]: Also, mention object structure that it sends to API

File: rollup.config.js (hunk 3 above)
Comment (kunalpanchal) [reply rollup.config.js]: We have used [create-react-library](https://www.npmjs.com/package/create-react-library) npm to build this, and it uses rollup internally. I plan to switch to webpack later sometimes.

File: dist/index.es.js (hunk 2 above)
Comment (kunalpanchal) [reply dist/index.es.js]: We will require /dist until we publish this to npmjs.org

File: README.md (hunk 1 above)
Comment (kunalpanchal) [reply README.md]: Done, updated the README.md.

**Pull request merged**